docs/loading/Flink-connector-starrocks.md

```diff
@@ -1,173 +1,647 @@
+
+    ```SQL
+    CREATE TABLE `page_uv` (
+        `page_id` INT,
+        `visit_date` TIMESTAMP,
+        `visit_user_id` BIGINT
+    ) WITH (
+        'connector' = 'starrocks',
+        'jdbc-url' = 'jdbc:mysql://127.0.0.1:9030',
+        'load-url' = '127.0.0.1:8030',
+        'database-name' = 'test',
+        'table-name' = 'page_uv',
+        'username' = 'root',
+        'password' = '',
+        'sink.properties.columns' = 'page_id,visit_date,visit_user_id,visit_users=to_bitmap(visit_user_id)'
+    );
+    ```
+
+3. Load data into Flink table in Flink SQL client
+
+    ```SQL
+    INSERT INTO `page_uv` VALUES
+       (1, CAST('2020-06-23 01:30:30' AS TIMESTAMP), 13),
+       (1, CAST('2020-06-23 01:30:30' AS TIMESTAMP), 23),
+       (1, CAST('2020-06-23 01:30:30' AS TIMESTAMP), 33),
+       (1, CAST('2020-06-23 02:30:30' AS TIMESTAMP), 13),
+       (2, CAST('2020-06-23 01:30:30' AS TIMESTAMP), 23);
+    ```
+
+4. Calculate page UVs from the StarRocks table in MySQL client.
+
+    ```SQL
+    MySQL [test]> SELECT `page_id`, COUNT(DISTINCT `visit_users`) FROM `page_uv` GROUP BY `page_id`;
+    +---------+-----------------------------+
+    | page_id | count(DISTINCT visit_users) |
+    +---------+-----------------------------+
+    |       2 |                           1 |
+    |       1 |                           3 |
+    +---------+-----------------------------+
+    2 rows in set (0.05 sec)
+    ```
+
+### Load data into columns of HLL type
+
+[`HLL`](../sql-reference/sql-statements/data-types/HLL) can be used for approximate count distinct, see [Use HLL for approximate count distinct](../using_starrocks/Using_HLL).
+
+Here we take the counting of UV as an example to show how to load data into columns of the `HLL` type.
+
+1. Create a StarRocks Aggregate table
+
+   In the database `test`, create an Aggregate table `hll_uv` where the column `visit_users` is defined as the `HLL` type and configured with the aggregate function `HLL_UNION`.
+
+    ```SQL
+    CREATE TABLE `hll_uv` (
+      `page_id` INT NOT NULL COMMENT 'page ID',
+      `visit_date` datetime NOT NULL COMMENT 'access time',
+      `visit_users` HLL HLL_UNION NOT NULL COMMENT 'user ID'
+    ) ENGINE=OLAP
+    AGGREGATE KEY(`page_id`, `visit_date`)
+    DISTRIBUTED BY HASH(`page_id`);
+    ```
+
+2. Create a Flink table in Flink SQL client
+
+   `visit_user_id` is `BIGINT` in Flink, and we want to load it to the column `visit_users` of StarRocks table. Note that for the Flink DDL
+    - define the `visit_user_id` instead of `visit_users` because Flink does not support `BITMAP`
+    - set the option `sink.properties.columns` to `page_id,visit_date,user_id,visit_users=hll_hash(visit_user_id)` which tells the connector the column mapping
+      between Flink table and StarRocks table. It uses the [`hll_hash`](../sql-reference/sql-functions/aggregate-functions/hll_hash)
+      function to convert the data of `BIGINT` type into `HLL` type.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/dbc8d44b-fa86-4d92-92fb-bd1a1099fff4)
At least capitalize the first letter and use a period at the sentence end.